ceph.spec.in

```diff
@@ -656,9 +656,11 @@ Requires:       python%{python3_pkgversion}-remoto
 Requires:       cephadm = %{_epoch_prefix}%{version}-%{release}
 %if 0%{?suse_version}
 Requires:       openssh
+Requires:	      python%{python3_pkgversion}-Jinja2
```

Comment:
this line contains a `\t` character. maybe substitute with spaces to make the diff align with line 658?

Reply:
Fixed. Unfortunately the tab/space is mixed everywhere in the file.